recipes/maxbin2/meta.yaml

```diff
@@ -0,0 +1,28 @@
+build:
+  number: 0
+
+package:
+  name: maxbin2
+  version: "2.2.1"
+source:
+  fn: MaxBin-2.2.1.tar.gz
+  url: http://downloads.jbei.org/data/microbial_communities/MaxBin/getfile.php?MaxBin-2.2.1.tar.gz
+  md5: 3748e8f65b5bf18f8bd341210724cc5c
+requirements:
+  build:
+    - gcc # [linux]
+    - llvm # [osx]
+  run:
+    - libgcc # [linux]
+    - perl
+    - bowtie2
+    - fraggenescan >=1.30
+    - hmmer
+    - idba
+test:
+  commands:
+    - MaxBin > /dev/null
```

Comment:
I did notice that the return code was 255. Usually I would test for installation using 
`MaxBin --help` or `MaxBin --version`, but the program only accepts real inputs. Any ideas?